webapp.yml

```diff
@@ -0,0 +1,92 @@
+# Description
+# ===========
+# This playbook creates 
+# - An Azure Linux Web app with Java container Tomcat 8.5.
+# - An Azure Traffic manager profile.
+# - Add the web site created as an endpoint of the Traffic Manager Profile.
+#
+# This playbook requires Ansible version >=2.7, or you could use role Azure.azure_preview_modules
+#
+
+- hosts: localhost
+  connection: local
+  vars:
+    app_name: testapp0917
+    location: eastus
+    linux_plan_name: testplan0917
+    traffic_manager_profile_name: profile0917
+    traffic_manager_endpoint_name: endpoint1
+
+  roles:
```

Comment:
i think I would comment out role, and say # uncomment that if not using ansible 2.7

Reply:
if comment out, it not working since 2.7 not released yet.

Reply:
yeah, that's why we can wait perhaps